src/base1/patternfly-overrides.less

```diff
@@ -1 +1,559 @@
+
+@btn-success-color:              #fff;
+@btn-success-bg:                 @brand-success;
+@btn-success-border:             @btn-success-bg;
+
+@btn-info-color:                 #fff;
+@btn-info-bg:                    @brand-info;
+@btn-info-border:                @btn-info-bg;
+
+@btn-warning-color:              #fff;
+@btn-warning-bg:                 @brand-warning;
+@btn-warning-border:             @btn-warning-bg;
+
+@btn-danger-color:               #fff;
+@btn-danger-bg:                  @brand-danger;
+@btn-danger-border:              @btn-danger-bg;
+
+@btn-link-disabled-color:        @gray-light;
+
+@line-height-base: 1.5;
+@font-size-base  : 16px;
+
+// Restyle inputs & dropdowns
+.input-group-addon,
+.bootstrap-select.btn-group .btn,
+.content-header-extra .dropdown-toggle,
+.dropdown-toggle,
+.form-control:not(.ct-select) {
+  background-image: none !important;
+  border-radius: 3px;
+  border-color: var(--pf-global--BorderColor--300);
+  border-bottom-color: var(--pf-global--Color--200);
+  color: var(--pf-global--Color--100);
+  box-shadow: none !important;
+  min-height: 2.25rem;
+
+  &:not(.bootstrap-select):not(.dropdown-form-control) {
+    padding: 1px 0.5rem;
+  }
+
+  .caret {
+    align-self: center;
+
+    &::before {
+      content: "\f0d7";
+    }
+  }
+
+  &::placeholder {
+    color: var(--pf-global--Color--dark-200);
+    font-style: normal;
+  }
+
+  &:active, &:focus, &:hover {
+    border-color: var(--pf-global--BorderColor--300);
+    border-bottom-color: var(--pf-global--primary-color--100);
+    color: var(--pf-global--Color--100);
+  }
+
+  &:focus {
+    // Take 1px from the padding and add it to the border
+    padding-bottom: 0;
+    border-bottom-width: 2px;
+  }
+
+  &:disabled {
+    background-color: var(--pf-global--disabled-color--300);
+    border-color: var(--pf-global--disabled-color--300);
+    color: var(--pf-global--disabled-color--100);
+    cursor: not-allowed;
+  }
+
+  .has-error &,
+  &.has-error {
+    position: relative;
+
+    &, &:hover, &:focus {
+      border-color: var(--pf-global--BorderColor--300);
+      border-bottom: 2px solid var(--pf-global--danger-color--100);
+    }
+  }
+}
+
+// Fix split dropdowns
+.input-group {
+  display: inline-flex;
+
+  .combobox-container & {
+    width: 100%;
+  }
+
+  > .form-control {
+    display: flex;
+    flex: auto;
+  }
+
+  > .input-group-addon,
+  > .dropdown,
+  > .dropdown-toggle {
+    display: flex;
+    width: auto;
+    align-items: stretch;
+  }
+
+  > .input-group-btn > .btn {
+    min-width: 2.5rem;
+  }
+}
+
+// Restyle buttons
+.btn {
+  padding: 0.375rem 1rem;
+
+  &:not(.dropdown-toggle):not(.dropdown):not(.bootstrap-select):not(.ct-select) {
+    background-image: none !important;
+    border-radius: 3px;
+
+    &:not(:active):not(:focus):not(:hover) {
+      box-shadow: none !important;
+    }
+
+    &:active, &:focus, &:hover {
+      border-color: var(--pf-global--primary-color--200);
```

Comment:
PF4 uses `--pf-global--primary-color--100` for the hover states of the secondary button here as far as I can see, so it doesn't change color on hover.

Reply:
https://www.patternfly.org/v4/documentation/core/components/button